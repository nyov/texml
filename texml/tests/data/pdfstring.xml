<TeXML>
  _/0189:_@ABYZ[_`abyz{_&#x44f;
  <spec cat="nl"/>
  <pdfstring>_/0189:_@ABYZ[_`abyz{_&#x44f;</pdfstring>
</TeXML>
