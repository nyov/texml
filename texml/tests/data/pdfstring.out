\_/0189:\_@ABYZ[\_`abyz\{\_\cyrchar\cyrya{}
\000\137\000\057\0000\0001\0008\0009\000\072\000\137\000\100\000A\000B\000Y\000Z\000\133\000\137\000\140\000a\000b\000y\000z\000\173\000\137\004\117
